src/serlo-editor/plugins/rows/components/rows-editor.tsx

```diff
@@ -53,14 +59,19 @@ export function RowsEditor(props: RowsProps) {
 
   const isDocumentEmpty = props.state.length === 0
 
+  const plugins = getPluginRegistry(
+    config.parentType,
+    editorStrings,
+    config.allowedPlugins
+  )
```

Comment:
I think this does not work for `serlo-editor-for-edusharing`. There we have additional plugins (for example the edusharing-asset) and define our own registry `serlo-editor-for-edusharing/src/frontend/plugins/registry.ts`. `getPluginRegistry` here cannot return the edusharing plugin. I guess we either need to add an entry for the edu-sharing asset also to the registry in `frontend` or provide an option to extend / overwrite the registry when calling `createRowsPlugin` like before. Unsure right now, what I would go for.

Reply:
> Should we provide the registry as a prop to the Editor like the plugins?

That would be great I think! At some point we could even change it so that we only pass one prop that contains everything.

Reply:
Great! I just tested this in `serlo-editor-for-edusharing` and it seems to work. However, I noticed a few weird things: 
- Plugin drag & drop does not work in the outermost scope. Within a box, it works. 
- Multimedia plugin has more padding and an outline when focused. 
I need to have a closer look into what is happening there.

Reply:
thanks for testing! 
I guess what you report is caused by my changes and does not have anything to do with the integration.

Reply:
@LarsTheGlidingSquirrel could you confirm that the multimedia plugin issue also happens on staging and is not related to this PR?

Reply:
drag & drop problm should be fixed now, good catch!